### PR TITLE
Fix build script

### PR DESCRIPTION
**Pull request by KorDum**
Fix build.gradlew
Added README.md

```version = version_major + '.' + version_minor + '.' + version_patch```
Where does the `version_patch` come from?

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -57,9 +60,9 @@ processResources
     // replace stuff in mcmod.info, nothing else
     from(sourceSets.main.resources.srcDirs) {
         include 'mcmod.info'
-                
-        // replace version and mcversion
-        expand 'version':project.version, 'mcversion':project.minecraft.version
+        expand 'version': project.version, 
+                'mcversion': project.minecraft.version,
+                'modid': archivesBaseName
```
Comment (marcin212) [inline build.gradle]: Should modid be "binnie-mods-1.11.2" ?

Comment (KorDum) [reply build.gradle]: Fixed, thanks!

File: build.gradle [hunk 2]
```diff
@@ -62,7 +62,7 @@ processResources
         include 'mcmod.info'
         expand 'version': project.version, 
                 'mcversion': project.minecraft.version,
-                'modid': archivesBaseName
+                'modid': "BinnieMods"
```
Comment (marcin212) [inline build.gradle]: IIRC, modid in 1.11.2 should be only lower case.

Comment (mezz) [reply build.gradle]: Yeah has to be lowercase. It's enforced by Forge.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+## Binnie mods
+
+All builds can be found on [Jenkins](https://ci.bymarcin.com/job/Binnie-Mods-MC1.11.2/)
```
Comment (mezz) [inline README.md]: The state of 1.11.2 is buggy and incomplete. This line should be removed, or a disclaimer added.

File: build.gradle (hunk 2 above)
Comment (KorDum) [reply build.gradle]: Fixed

File: README.md (hunk 3 above)
Comment (KorDum) [reply README.md]: Temporary removed

**Pull request merged**